docs/get-started/installing-cardano-wallet.md

```diff
@@ -100,4 +100,6 @@ Congratulations, you have successfully installed `cardano-wallet` into your Linu
 
 :::important
 Currently, the **Windows** installation guide is still in-progress. In the meantime we recommend using [WSL (Windows Subsystem for Linux)](https://docs.microsoft.com/en-us/windows/wsl/) to get a Linux environment on-top of Windows. Once you have that installed you can use the [Linux](#linux) guide to install and run `cardano-node` within **WSL**.
+
+Optionally, you can download the latest precompiled Windows binary from the [`cardano-wallet`](https://github.com/cardano-foundation/cardano-wallet) github repository.
```

Comment:
```suggestion
Optionally, you can download the latest precompiled Windows binary from the GitHub repository:  [`cardano-wallet` Releases](https://github.com/cardano-foundation/cardano-wallet/releases)
```
I don't feel this comment is necessary, but as someone who never uses the Windows platform for anything it would be unfair of me to rule this.  If it is added I believe it should be in the same form as the link above... to avoid suggesting the download place is different for Windows.